tools/argdist.py

```diff
@@ -684,9 +686,11 @@ def _attach(self):
 
         def _main_loop(self):
                 count_so_far = 0
+                seconds = 0
                 while True:
                         try:
-                                sleep(self.args.interval)
+                                sleep(int(self.args.interval))
```

Comment:
I think this `int` casting might not be necessary (same for the ones below). argparse should handle that for us already here: https://github.com/iovisor/bcc/pull/1768/files#diff-080a37c8911a2d34cbff11a2f1ca1255R612 😄